api/tests/unit/domain/services/mail-service_test.js

```diff
@@ -193,84 +190,104 @@ describe('Unit | Service | MailService', () => {
 
   describe('#sendResetPasswordDemandEmail', () => {
 
+    const temporaryKey = 'token';
+
+    const translationsMapping = {
+      fr: mainTranslationsMapping.fr['reset-password-demand-email'],
+      en: mainTranslationsMapping.en['reset-password-demand-email'],
+    };
+
     context('when provided passwordResetDemandBaseUrl is not production', () => {
 
       it('should call mailer', async () => {
         // given
-        const fakeTemporaryKey = 'token';
         const locale = 'fr-fr';
         const domainFr = 'pix.fr';
 
         const expectedOptions = {
           from: senderEmailAddress,
           fromName: 'PIX - Ne pas répondre',
           to: userEmailAddress,
-          subject: 'Demande de réinitialisation de mot de passe PIX',
+          subject: translationsMapping.fr.subject,
           template: 'test-password-reset-template-id',
           variables: {
-            resetUrl: `https://app.${domainFr}/changer-mot-de-passe/${fakeTemporaryKey}`,
+            locale,
+            ...translationsMapping.fr.params,
             homeName: `${domainFr}`,
             homeUrl: `https://${domainFr}`,
-            locale,
+            resetUrl: `https://app.${domainFr}/changer-mot-de-passe/${temporaryKey}`,
+            helpdeskURL: 'https://support.pix.fr/support/tickets/new',
           },
         };
 
         // when
-        await mailService.sendResetPasswordDemandEmail(userEmailAddress, locale, fakeTemporaryKey);
+        await mailService.sendResetPasswordDemandEmail({
+          email: userEmailAddress,
+          locale,
+          temporaryKey,
+        });
 
         // then
         expect(mailer.sendEmail).to.have.been.calledWithExactly(expectedOptions);
       });
 
       it('should call mailer with locale fr', async () => {
```

Comment:
**suggestion**
L'assertion qui correspond au nom du test `should call mailer with locale fr `serait 

`mailer.sendEmail.firstCall.args[0].locale = 'fr;`

Je pense qeu ce n'est pas l'intention et propose plutôt de renommer le test 
```suggestion
      it('should call mailer with localized options', async () => {
```

Reply:
On veut faire la distinction `fr` et `en`.
L'intention : appeler le mailer avec les bons params `fr` ou `en`.

Reply:
Dans ce cas, l'assertion correspondante est `mailer.sendEmail.firstCall.args[0].locale = 'fr;

Reply:
Je précise

Soit faire référence au nom de méthode et de paramètre
```suggestion
      it('should call mailer.sendEmail with localized options', async () => {
```

Soit être métier
```suggestion
      it('should call mailer with localized data, async () => {
```

Mais en aucun cas ` it('should call mailer with locale fr', async () => {` qui fait penser qu'il existe un truc nommé `locale`
C'était ce qui se passait avant où on envoyait la locale et on traduisait dans sendInBlue, mais ça c'est du passé !

Reply:
Done